### PR TITLE
refactor(transactions) Switch `AccountSet` to new transaction scheme

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

Moves `AccountSet` to the new transaction schema, along with hooks, typescript, and more tests.

This also adds NFTokenMinter description to detail tab.

### Context of Change

Part of #200

### Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

### TypeScript/Hooks Update

- [x] Updated files to React Hooks
- [x] Updated files to TypeScript

### CODE REVIEW

File: src/containers/shared/components/Transaction/AccountSet/parser.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { AccountSet } from './types'
+
+export const parser = (tx: AccountSet): AccountSet => tx
```
Comment (mvadari) [inline src/containers/shared/components/Transaction/AccountSet/parser.ts:3]: ++ simplifying/eliminating the parser

File: src/containers/shared/components/Transaction/AccountSet/Description.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import { ACCOUNT_FLAGS } from '../../../transactionUtils'
+import DomainLink from '../../DomainLink'
+import { TransactionDescriptionProps } from '../types'
+import { AccountSet } from './types'
+import { Account } from '../../Account'
+
+export const Description = ({
```
Comment (mvadari) [inline src/containers/shared/components/Transaction/AccountSet/Description.tsx:9]: Is this even needed? Seems like this is the same as what's in `Simple`

Comment (ckniffen) [reply src/containers/shared/components/Transaction/AccountSet/Description.tsx:9]: The types of values shown are the same but it prints them out into sentences.  For example NFTokenMinter it shows

> It sets rXMART8usFd5kABXCayoP6ZfB35b4v43t as the authorized minter for this account

**Review by JST5000 — APPROVED** [review]
LGTM

**Review by mvadari — APPROVED** [review]

**Review by mvadari — APPROVED** [review]

**Pull request merged**